src/index.html

```diff
@@ -16,7 +16,7 @@ <h1>Today's To Do</h1><i class="fa-solid fa-rotate"></i>
      <hr>
```

Comment:
- [ ] I can't see the button to add a task, Kindly make sure that you add that button and then ask for re-review.
![image](https://user-images.githubusercontent.com/71874425/172023977-aa476643-5627-4b62-9cdf-d6b348dc3a19.png)
- [ ] Also, to fit the design provide by the framework you should use for this project, I'll suggest you to remove the black border shadow.